### PR TITLE
All master data attributes can be changed

### PR DESCRIPTION
**Pull request by kft**
## Description
This adds support for updating and changing all master data attributes. Currently, no reject message is generated if invalid values are provided. This will be added later on.

## References

* #551

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/BusinessRequestProcessing/Protobuf/Contracts/Contract.proto [hunk 1]
```diff
@@ -27,43 +27,44 @@ message Address {
...
-  string municipalityCode = 10;
+  google.protobuf.StringValue municipalityCode = 10;
   google.protobuf.StringValue cityNameSubDivision = 11;
   google.protobuf.StringValue geoInfoReference = 12;
 }
 
 message MasterDataDocument {
   string gsrnNumber = 1;
-  string typeOfMeteringPoint = 2;
+  google.protobuf.StringValue typeOfMeteringPoint = 2;
```
Comment (Korgath) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/BusinessRequestProcessing/Protobuf/Contracts/Contract.proto:40]: Can typeOfMeteringPoint be null ?

File: source/Energinet.DataHub.MeteringPoints.Application/Create/CreateMeteringPoint.cs [hunk 2]
```diff
@@ -32,8 +32,8 @@ public record CreateMeteringPoint(
             string GsrnNumber = "",
             string MeteringMethod = "",
             string MeterReadingOccurrence = "",
-            int? MaximumCurrent = 0,
-            int? MaximumPower = 0,
+            string? MaximumCurrent = null,
```
Comment (Korgath) [inline source/Energinet.DataHub.MeteringPoints.Application/Create/CreateMeteringPoint.cs:35]: Why the change from int to string?

File: source/Energinet.DataHub.MeteringPoints.Application/MarketDocuments/MasterDataDocument.cs [hunk 3]
```diff
@@ -26,25 +26,25 @@ public record MasterDataDocument(
...
             string? RoomIdentification = null,
             bool? IsActualAddress = null,
             string GsrnNumber = "",
             string TypeOfMeteringPoint = "",
             string? MeteringMethod = null,
-            string MeterReadingOccurrence = "",
-            int? MaximumCurrent = null,
-            int? MaximumPower = null,
+            string? MeterReadingOccurrence = null,
+            string? MaximumCurrent = null,
```
Comment (Korgath) [inline source/Energinet.DataHub.MeteringPoints.Application/MarketDocuments/MasterDataDocument.cs:39]: Same as above

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/BusinessRequestProcessing/Protobuf/Contracts/Contract.proto (hunk 1 above)
Comment (kft) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/BusinessRequestProcessing/Protobuf/Contracts/Contract.proto:40]: Yes
